test/reactionsystem_core/reactionsystem.jl

```diff
@@ -356,7 +356,7 @@ let
     oprob1 = ODEProblem(osys, u0map, tspan, pmap)
     sts = [B, D, E, C]
     syms = [:B, :D, :E, :C]
-    ofun = ODEFunction(f!; syms)
+    ofun = ODEFunction(f!; sys = ModelingToolkit.SymbolCache(syms))
```

Comment:
Not too familiar with this one, simply copied what the deprecation warning said.